Pod/Classes/LSTracer.m

```diff
@@ -373,23 +386,23 @@ - (void)flush:(void (^)(NSError * _Nullable error))doneCallback {
         req.timestampOffsetMicros = m_clockState.offsetMicros;
         m_pendingProtoSpans = [NSMutableArray<LSPBSpan*> array];
         m_lastFlush = now;
-        
+
         m_bgTaskId = [[UIApplication sharedApplication]
                       beginBackgroundTaskWithName:@"com.lightstep.flush"
                       expirationHandler:^{
-                          cleanupBlock([NSError errorWithDomain:LSErrorDomain code:LSBackgroundTaskError userInfo:nil]);
+                          cleanupBlock(true, [NSError errorWithDomain:LSErrorDomain code:LSBackgroundTaskError userInfo:nil]);
```

Comment:
It would be bad if the expirationHandler were called before `beginBackgroundTaskWithName:expirationHandler:` returns... that's certainly not the implied semantics when I read the docs, but it's not explicitly ruled out.